src/mlir/generator.h

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ast/ast.h"
+#include "mlir/IR/Builders.h"
+#include "mlir/IR/Function.h"
+#include "mlir/IR/MLIRContext.h"
+#include "mlir/IR/Module.h"
+#include "mlir/Target/LLVMIR.h"
+#include "symbol.h"
+
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+
+#include <peglib.h>
+#include <string>
+
+namespace mlir::verona
+{
+  // MLIR Generator.
```

Comment:
Please can you make doc comments something that a doc comment extractor can spot?  I prefer `/** */`, LLVM likes `///`.